tools/monitoring/workflow/alerting.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package workflow
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"github.com/knative/test-infra/shared/gcs"
+	"github.com/knative/test-infra/tools/monitoring/config"
+	"github.com/knative/test-infra/tools/monitoring/log_parser"
+	"github.com/knative/test-infra/tools/monitoring/prowapi"
+	"github.com/knative/test-infra/tools/monitoring/subscriber"
+)
+
+const yamlURL = "https://raw.githubusercontent.com/knative/test-infra/master/tools/monitoring/config/sample.yaml"
+
+// RunAlerting start the alerting workflow
+func RunAlerting(client *subscriber.Client) {
```

Comment:
what do you mean by the alerting workflow? Why doesn't this stay in the alert pkg?

Reply:
Moved to alert package.